sdk/servicebus/azure-servicebus/azure/servicebus/_common/message.py

```diff
@@ -240,27 +240,39 @@ def time_to_live(self, value):
             self.header.time_to_live = int(value) * 1000
 
     @property
-    def body(self):
-        # type: () -> Union[bytes, Generator[bytes]]
-        """The body of the Message.
+    def scheduled_enqueue_time_utc(self):
+        # type: () -> Optional[datetime.datetime]
+        """Get or set the utc scheduled enqueue time to the message.
+        This property can be used for scheduling when sending a message through `ServiceBusSender.send` method.
```

Comment:
So this is worded well, so don't take this as an issue with how you wrote it, but more what it communicates:  I'm wondering if we should say something like (for the third sentence; "or you could use..."):  "If cancelling scheduled messages is required, you should use the ServiceBusSender.schedule method, which returns sequence numbers that can be used for future cancellation".

This is a total nitpick simply meant to make more clear the deliniation between what capabilities you get here vs there.

Reply:
your words are better, updated.

but I couldn't understand "returns none if scheduled_enqueue_time has not been populated.", it user use `sender.schedule`, the method requires user to pass in a "schedule_enqueue_time_utc", and that time shall be populated anyway which would always return sequence number.

Reply:
So I worded the latter part badly; my key was to emphasize that schedule_enqued_time_utc is None if not set.

May be unnecessary in hindsight, it feels bikesheddy as I say it; so feel free to use your judgement.

Reply:
updated, thanks for the words!